examples/src/loadfile.rs

```diff
@@ -0,0 +1,29 @@
+use std::{fs::read_to_string};
+
+use boa::{
+    exec::Interpreter,
+    forward,
+    realm::Realm
+};
+
+pub fn run(){
+    let js_file_path = "./scripts/helloworld.js";
+    let buffer = read_to_string(js_file_path);
+
+    if buffer.is_err(){
+        println!("Error: {}", buffer.unwrap_err());
+        return;
+    }
```

Comment:
Make it more idiomatic.
```suggestion
    if let Err(msg) = buffer {
        println!("Error: {}", msg);
        return;
    }
```